src/app/pages/apps-old/pod-logs/pod-logs.component.html

```diff
@@ -32,6 +32,10 @@
     class="pod-logs"
     [ngStyle]="{ 'font-size': fontSize + 'px' }"
   >
+    <div *ngIf="areLoadingPodLogs" class="log-row">
+      <span  class="no-logs">{{ 'No logs' }}</span>
```

Comment:
You should instead do something like
`<div *ngIf="isLoadingPodLogs">...show loading spinner....</div> <div *ngIf="!isLoadingPodLogs"> <div *ngIf="!podLogs.length">{{ 'No logs yet' | translate }}</div><div *ngIf="podLogs.length">...show logs...</div></div>`